core/o11y/src/lib.rs

```diff
@@ -100,12 +100,12 @@ fn is_terminal() -> bool {
 ///
 /// ```rust
 /// let filter = near_o11y::EnvFilterBuilder::from_env().finish().unwrap();
-/// let _subscriber = near_o11y::default_subscriber(filter, near_o11y::ColorOutput::Auto);
+/// let _subscriber = near_o11y::default_subscriber(filter, &near_o11y::ColorOutput::Auto);
 /// near_o11y::tracing::info!(message = "Still a lot of work remains to make it proper o11y");
 /// ```
```

Comment:
Could you augment the documentation to specify that this must be called within an async runtime? I think ideally this would be enforced by the API (i.e. make this function `async` or make it return an `impl Future` or make it take a `tokio::Runtime` as an argument, or...

So that it isn't possible to forget it.

Reply:
Made the function `async`.
Passing a `Runtime` would be ideal, but can't do that with `opentelemetry` API.